crates/re_viewer/src/ui/visible_history.rs

```diff
@@ -84,64 +84,65 @@ pub fn visible_history_ui(
 
     let re_ui = ctx.re_ui;
 
-    re_ui
-        .collapsing_header(ui, "Visible Time Range", true, |ui| {
-            ui.horizontal(|ui| {
-                re_ui
-                    .radio_value(ui, &mut visible_history_prop.enabled, false, "Default")
-                    .on_hover_text(if is_space_view {
-                        "Default Visible Time Range settings for this kind of Space View"
-                    } else {
-                        "Visible Time Range settings inherited from parent Group(s) or enclosing \
+    let sequence_timeline = matches!(ctx.rec_cfg.time_ctrl.timeline().typ(), TimeType::Sequence);
```

Comment:
Would have made the following code much easier to read without knowing the context:
```suggestion
    let is_sequence_timeline = matches!(ctx.rec_cfg.time_ctrl.timeline().typ(), TimeType::Sequence);